packages/@sanity/desk-tool/src/panes/document/statusBar/sparkline/PublishStatus.tsx

```diff
@@ -1,25 +1,24 @@
 /* eslint-disable complexity */
```

Comment:
This eslint rule override can probably be removed.

Reply:
Still gets the eslint error. But I found some ways to reduce complexity so that the rule can be removed!